Packages.props

```diff
@@ -35,7 +35,7 @@
 		<PackageReference Update="Microsoft.SqlServer.Assessment" Version="[1.1.17]" />
 		<PackageReference Update="Microsoft.SqlServer.Migration.Assessment" Version="1.0.20230907.105" />
 		<PackageReference Update="Microsoft.SqlServer.Migration.Logins" Version="1.0.20230407.56" />
-		<PackageReference Update="Microsoft.SqlServer.Management.QueryStoreModel" Version="163.26.1" />
+		<PackageReference Update="Microsoft.SqlServer.Management.QueryStoreModel" Version="163.55.1" />
```

Comment:
Checked in nuget packages 😭

Reply:
Is an unlisted nuget package preferred?  I _suppose_ it could be listed publicly, but that implies that it's meant to be consumed by external devs.

Reply:
*shrug* We push internal stuff to nuget, so it seems better for everyone to just bite the bullet and make it "official" (with a note about it being intended for internal use to avoid people complaining it doesn't work like they expect). If someone wants to try and use this then go for it I say.

(I understand this stuff comes with engineering overhead to set up the publishing pipelines and whatnot - but TBH that's well worth the cost)

Reply:
Discussed offline-but-technically-it-was-online-since-we-were-communicating-remotely-and-digitally-over-the-internet.

Not a blocking issue, and something I can pursue separately.